src/Details/Colors/Color.react.js

```diff
@@ -0,0 +1,24 @@
+var React = require('react');
+
+var Color = React.createClass({
+
+  render: function() {
+    console.log('Rendering Detail Color');
+    var divStyle = {
+      backgroundColor: this.props.color
+    };
+    var self = this;
+    var msg = function() {
+      return {__html: self.props.message + self.props.color };
+    };
+    
+    return (
+      <div className="details__color" style={divStyle}>
+        <div className="details__colorInner" dangerouslySetInnerHTML={msg()}></div>
```

Comment:
oh so you found `dangerouslySetInnerHTML`. I'm curious as to why you thought you had to do this? Also, could you explain why `msg` is a function and not a static value, as it appears it could be?